extensions/import/package.json

```diff
@@ -71,7 +71,7 @@
 		}
 	},
 	"dependencies": {
-		"dataprotocol-client": "github:Microsoft/sqlops-dataprotocolclient#1.2.2",
+		"dataprotocol-client": "github:Microsoft/sqlops-dataprotocolclient#0.3.0",
```

Comment:
Hm, have you merged in the latest from main? These changes shouldn't be here.

Reply:
Fixed commit tree.